docs/core/event_handler/api_gateway.md

```diff
@@ -1027,43 +1027,42 @@ When necessary, you can set a prefix when including a router object. This means
 
 #### Sample layout
 
-!!! info "We use ALB to demonstrate that the UX remains the same"
-
-This sample project contains an Users function with two distinct set of routes, `/users` and `/health`. The layout optimizes for code sharing, no custom build tooling, and it uses [Lambda Layers](../../index.md#lambda-layer) to install Lambda Powertools.
+This sample project contains a Users function with two distinct set of routes, `/users` and `/health`. The layout optimizes for code sharing, no custom build tooling, and it uses [Lambda Layers](../../index.md#lambda-layer) to install Lambda Powertools.
 
 === "Project layout"
 
 
-    ```python hl_lines="6 8 10-13"
+    ```python hl_lines="1 8 10 12-15"
     .
-    ├── Pipfile                   # project app & dev dependencies; poetry, pipenv, etc.
+    ├── Pipfile                    # project app & dev dependencies; poetry, pipenv, etc.
```

Comment:
Shall we update to poetry pyproject.tomo since we're gonna update the sample cookiecutter already?

Reply:
@heitorlessa - currently I like `Pipefile` as this is the default for SAM cli and Amplify templates. However I am cool to switch to Poetry

Reply:
So let's keep as-is until we migrate others.